src/test/testUtils.js

```diff
@@ -44,8 +44,8 @@ export const renderWithReduxRouter = async (
       element: <LandingPage />,
     },
     {
-      path: 'insights/image-builder/imagewizard/:composeId?',
-      element: <CreateImageWizard />,
+      path: 'insights/image-builder/imagewizard/import',
+      element: <ImportImageWizard />,
```

Comment:
Shouldn't this be path to the composeId using V2?

```suggestion
      path: 'insights/image-builder/imagewizard/:composeId',
      element: <CreateImageWizard />,
```

with V2 import instead of Import path?

Reply:
the only place where we use this function for mounting currently is the import wizard.
We use `renderCustomRoutesWithReduxRouter` elsewhere, which I'd propose to fix in a follow-up.

Reply:
I see, yeah makes sense then :sweat_smile: